### PR TITLE
treewide: fix typos and add manpage interlinking.

### PR DESCRIPTION
**Pull request by torstehu**

### CODE REVIEW

File: man/sd_bus_get_fd.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
         project='die-net'><refentrytitle>select</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
       <citerefentry
         project='die-net'><refentrytitle>poll</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
-      or similar functions to wait for incmming messages.
+      or similar functions to wait for incomming messages.
```
Comment (martinpitt) [inline man/sd_bus_get_fd.xml]: While we are at it, please _really_ fix it to "incoming" :-)

**Comment by martinpitt** [comment]
LGTM, just one more typo. Can you please amend/force-push this? Thanks!

**Comment by torstehu** [comment]
Thank you! Rebased, fixed and force-pushed.

**Comment by martinpitt** [comment]
Thanks!

**Pull request merged**